src/nrtk_explorer/app/embeddings.py

```diff
@@ -20,6 +22,41 @@
 from trame.app import get_server, asynchronous
 
 
+IdToImage = Dict[str, Image.Image]
+
+
+@TrameApp()
+class TransformedImages:
+    def __init__(self, server):
+        self.server = server
+        self.transformed_images: IdToImage = {}
+
+    def emit_update(self):
+        self.server.controller.update_transformed_images(self.transformed_images)
+
+    def add_images(self, dataset_id_to_image: IdToImage):
+        self.transformed_images.update(dataset_id_to_image)
+        self.emit_update()
+
+    @change("dataset_ids")
```

Comment:
Just making a note, but don't change it in thie PR: I noticed that we have been using `dataset_ids` to store a list of image ids. I think we should change the name to make the intent less confusing

Reply:
Open to naming ideas.  The totaly in my head convention was "dataset id"s mapped to a key in the COCO json, and "image ids" were keys for the original/transformed image on the state. 
https://github.com/Kitware/nrtk-explorer/blob/main/src/nrtk_explorer/app/images/image_ids.py#L26-L29